  INFO root [2019-09-04 20:33:03,899] - 
  INFO root [2019-09-04 20:33:03,900] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-04 20:33:03,901] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-04 20:33:03,902] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-04 20:33:03,902] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-04 20:33:03,926] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:30099 )
  INFO root [2019-09-04 20:33:04,103] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:30099, SOMAXCONN=5.
  WARN root [2019-09-04 20:33:04,104] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-04 20:33:04,108] - ---- interfaces Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: 96E445E56B9E460E561358C2D6E9F43B. Config: Debug X86. Built: 09:54:37 May  8 2019. AppID: 3000. UID: 2082. PID: 14544 ----
  INFO root [2019-09-04 20:33:04,108] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-04 20:33:04,108] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-04 20:33:04,109] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-04 20:33:04,317] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-04 20:33:04,318] - Script::install(): KBExtra is successfully!
S_INFO root [2019-09-04 20:33:04,362] - onInterfaceAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-04 20:33:04,364] - TelnetServer server is running on port 33000
  INFO root [2019-09-04 20:33:04,844] - ---- interfaces is running ----
 DEBUG root [2019-09-04 20:33:04,847] - Components::process(): Request for the process of identity...
  WARN root [2019-09-04 20:33:12,557] - d:\kbengine\kbe\src\lib\network\event_dispatcher.cpp:153: Profile callSystemTimers took 0.26 seconds
  INFO root [2019-09-04 20:33:13,677] - ServerApp::onRegisterNewApp: uid:2082, username:error_name, componentType:dbmgr, componentID:4000, globalorderID=-1, grouporderID=-1, intaddr:192.168.56.1, intport:56213, extaddr:nonsupport, extport:0,  from 127.0.0.1:56221/0/0/0.
  INFO root [2019-09-04 20:33:13,814] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-04 20:33:13,814] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  WARN root [2019-09-04 20:33:13,822] - d:\kbengine\kbe\src\lib\network\channel.cpp:1023: Profile processRecvMessages took 0.43 seconds
  INFO root [2019-09-04 20:33:14,304] - Components::process(): Found all the components!
  WARN root [2019-09-04 20:33:33,153] - TCPPacketReceiver::processPendingEvents(127.0.0.1:56221): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
  INFO root [2019-09-04 20:33:33,189] - Components::removeComponentByChannel: dbmgr : 4000, Normal exit!
  INFO root [2019-09-04 20:33:34,708] - 
  INFO root [2019-09-04 20:33:34,709] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-04 20:33:34,710] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-04 20:33:34,711] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-04 20:33:34,711] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-04 20:33:34,727] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:30099 )
  INFO root [2019-09-04 20:33:34,728] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:30099, SOMAXCONN=5.
  WARN root [2019-09-04 20:33:34,728] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-04 20:33:34,733] - ---- interfaces Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: 96E445E56B9E460E561358C2D6E9F43B. Config: Debug X86. Built: 09:54:37 May  8 2019. AppID: 3000. UID: 2082. PID: 21644 ----
  INFO root [2019-09-04 20:33:34,734] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-04 20:33:34,737] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-04 20:33:34,737] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-04 20:33:34,953] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-04 20:33:34,953] - Script::install(): KBExtra is successfully!
S_INFO root [2019-09-04 20:33:34,987] - onInterfaceAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-04 20:33:34,988] - TelnetServer server is running on port 33000
  INFO root [2019-09-04 20:33:35,381] - ---- interfaces is running ----
 DEBUG root [2019-09-04 20:33:35,381] - Components::process(): Request for the process of identity...
  INFO root [2019-09-04 20:33:41,487] - ServerApp::onRegisterNewApp: uid:2082, username:error_name, componentType:dbmgr, componentID:4000, globalorderID=-1, grouporderID=-1, intaddr:192.168.56.1, intport:56263, extaddr:nonsupport, extport:0,  from 127.0.0.1:56266/0/0/0.
  INFO root [2019-09-04 20:33:41,487] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-04 20:33:41,488] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-04 20:33:42,513] - Components::process(): Found all the components!
  WARN root [2019-09-04 20:34:29,880] - TCPPacketReceiver::processPendingEvents(127.0.0.1:56266): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
  INFO root [2019-09-04 20:34:29,883] - Components::removeComponentByChannel: dbmgr : 4000, Normal exit!
  INFO root [2019-09-04 20:34:31,564] - 
  INFO root [2019-09-04 20:34:31,572] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-04 20:34:31,573] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-04 20:34:31,574] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-04 20:34:31,574] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-04 20:34:31,628] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:30099 )
  INFO root [2019-09-04 20:34:31,628] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:30099, SOMAXCONN=5.
  WARN root [2019-09-04 20:34:31,628] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-04 20:34:31,635] - ---- interfaces Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: 96E445E56B9E460E561358C2D6E9F43B. Config: Debug X86. Built: 09:54:37 May  8 2019. AppID: 3000. UID: 2082. PID: 20664 ----
  INFO root [2019-09-04 20:34:31,640] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-04 20:34:31,641] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-04 20:34:31,641] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-04 20:34:31,887] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-04 20:34:31,887] - Script::install(): KBExtra is successfully!
S_INFO root [2019-09-04 20:34:31,914] - onInterfaceAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-04 20:34:31,916] - TelnetServer server is running on port 33000
  INFO root [2019-09-04 20:34:32,282] - ---- interfaces is running ----
 DEBUG root [2019-09-04 20:34:32,283] - Components::process(): Request for the process of identity...
  INFO root [2019-09-04 20:34:38,399] - ServerApp::onRegisterNewApp: uid:2082, username:error_name, componentType:dbmgr, componentID:4000, globalorderID=-1, grouporderID=-1, intaddr:192.168.56.1, intport:56312, extaddr:nonsupport, extport:0,  from 127.0.0.1:56318/0/0/0.
  INFO root [2019-09-04 20:34:38,399] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-04 20:34:38,399] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-04 20:34:39,414] - Components::process(): Found all the components!
  WARN root [2019-09-04 20:34:56,425] - TCPPacketReceiver::processPendingEvents(127.0.0.1:56318): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
  INFO root [2019-09-04 20:34:56,428] - Components::removeComponentByChannel: dbmgr : 4000, Normal exit!
  INFO root [2019-09-04 20:34:58,095] - 
  INFO root [2019-09-04 20:34:58,095] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-04 20:34:58,096] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-04 20:34:58,096] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-04 20:34:58,096] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-04 20:34:58,113] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:30099 )
  INFO root [2019-09-04 20:34:58,114] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:30099, SOMAXCONN=5.
  WARN root [2019-09-04 20:34:58,114] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-04 20:34:58,120] - ---- interfaces Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: 96E445E56B9E460E561358C2D6E9F43B. Config: Debug X86. Built: 09:54:37 May  8 2019. AppID: 3000. UID: 2082. PID: 9040 ----
  INFO root [2019-09-04 20:34:58,120] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-04 20:34:58,121] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-04 20:34:58,121] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-04 20:34:58,347] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-04 20:34:58,347] - Script::install(): KBExtra is successfully!
S_INFO root [2019-09-04 20:34:58,373] - onInterfaceAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-04 20:34:58,374] - TelnetServer server is running on port 33000
  INFO root [2019-09-04 20:34:58,753] - ---- interfaces is running ----
 DEBUG root [2019-09-04 20:34:58,753] - Components::process(): Request for the process of identity...
  INFO root [2019-09-04 20:35:04,870] - ServerApp::onRegisterNewApp: uid:2082, username:error_name, componentType:dbmgr, componentID:4000, globalorderID=-1, grouporderID=-1, intaddr:192.168.56.1, intport:56359, extaddr:nonsupport, extport:0,  from 127.0.0.1:56362/0/0/0.
  INFO root [2019-09-04 20:35:04,871] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-04 20:35:04,871] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-04 20:35:05,873] - Components::process(): Found all the components!
  WARN root [2019-09-04 20:35:51,040] - TCPPacketReceiver::processPendingEvents(127.0.0.1:56362): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
  INFO root [2019-09-04 20:35:51,047] - Components::removeComponentByChannel: dbmgr : 4000, Normal exit!
  INFO root [2019-09-04 20:56:08,476] - 
  INFO root [2019-09-04 20:56:08,476] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-04 20:56:08,477] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-04 20:56:08,477] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-04 20:56:08,477] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-04 20:56:08,495] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:30099 )
  INFO root [2019-09-04 20:56:08,496] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:30099, SOMAXCONN=5.
  WARN root [2019-09-04 20:56:08,496] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-04 20:56:08,500] - ---- interfaces Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: 96E445E56B9E460E561358C2D6E9F43B. Config: Debug X86. Built: 09:54:37 May  8 2019. AppID: 3000. UID: 2082. PID: 17324 ----
  INFO root [2019-09-04 20:56:08,500] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-04 20:56:08,500] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-04 20:56:08,501] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-04 20:56:08,773] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-04 20:56:08,773] - Script::install(): KBExtra is successfully!
S_INFO root [2019-09-04 20:56:08,802] - onInterfaceAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-04 20:56:08,804] - TelnetServer server is running on port 33000
  INFO root [2019-09-04 20:56:09,226] - ---- interfaces is running ----
 DEBUG root [2019-09-04 20:56:09,227] - Components::process(): Request for the process of identity...
  INFO root [2019-09-04 20:56:15,353] - ServerApp::onRegisterNewApp: uid:2082, username:error_name, componentType:dbmgr, componentID:4000, globalorderID=-1, grouporderID=-1, intaddr:192.168.56.1, intport:56632, extaddr:nonsupport, extport:0,  from 127.0.0.1:56641/0/0/0.
  INFO root [2019-09-04 20:56:15,354] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-04 20:56:15,354] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-04 20:56:16,379] - Components::process(): Found all the components!
  WARN root [2019-09-04 20:56:50,285] - TCPPacketReceiver::processPendingEvents(127.0.0.1:56641): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
  INFO root [2019-09-04 20:56:50,291] - Components::removeComponentByChannel: dbmgr : 4000, Normal exit!
  INFO root [2019-09-04 20:57:33,857] - 
  INFO root [2019-09-04 20:57:33,858] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-04 20:57:33,859] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-04 20:57:33,859] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-04 20:57:33,859] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-04 20:57:33,879] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:30099 )
  INFO root [2019-09-04 20:57:33,879] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:30099, SOMAXCONN=5.
  WARN root [2019-09-04 20:57:33,879] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-04 20:57:33,884] - ---- interfaces Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: 96E445E56B9E460E561358C2D6E9F43B. Config: Debug X86. Built: 09:54:37 May  8 2019. AppID: 3000. UID: 2082. PID: 21592 ----
  INFO root [2019-09-04 20:57:33,886] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-04 20:57:33,888] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-04 20:57:33,888] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-04 20:57:34,082] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-04 20:57:34,082] - Script::install(): KBExtra is successfully!
S_INFO root [2019-09-04 20:57:34,110] - onInterfaceAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-04 20:57:34,111] - TelnetServer server is running on port 33000
  INFO root [2019-09-04 20:57:34,487] - ---- interfaces is running ----
 DEBUG root [2019-09-04 20:57:34,487] - Components::process(): Request for the process of identity...
  INFO root [2019-09-04 20:57:39,345] - 
  INFO root [2019-09-04 20:57:39,345] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-04 20:57:39,346] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-04 20:57:39,346] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-04 20:57:39,347] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-04 20:57:39,366] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:30099 )
  INFO root [2019-09-04 20:57:39,366] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:30099, SOMAXCONN=5.
  WARN root [2019-09-04 20:57:39,367] - invalid UID(0) <= 0, please check UID for environment! automatically set to 2082.
  INFO root [2019-09-04 20:57:39,385] - ---- interfaces Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: 96E445E56B9E460E561358C2D6E9F43B. Config: Debug X86. Built: 09:54:37 May  8 2019. AppID: 3000. UID: 2082. PID: 6184 ----
  INFO root [2019-09-04 20:57:39,387] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-04 20:57:39,388] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-04 20:57:39,389] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-04 20:57:39,593] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-09-04 20:57:39,593] - Script::install(): KBExtra is successfully!
S_INFO root [2019-09-04 20:57:39,620] - onInterfaceAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-09-04 20:57:39,621] - TelnetServer server is running on port 33000
  INFO root [2019-09-04 20:57:40,001] - ---- interfaces is running ----
 DEBUG root [2019-09-04 20:57:40,001] - Components::process(): Request for the process of identity...
  INFO root [2019-09-04 20:57:46,120] - ServerApp::onRegisterNewApp: uid:2082, username:error_name, componentType:dbmgr, componentID:4000, globalorderID=-1, grouporderID=-1, intaddr:192.168.56.1, intport:56710, extaddr:nonsupport, extport:0,  from 127.0.0.1:56715/0/0/0.
  INFO root [2019-09-04 20:57:46,120] - Components::addComponent: reset orderLog, uid=2082!
  INFO root [2019-09-04 20:57:46,121] - Components::addComponent[dbmgr], uid=2082, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-04 20:57:47,149] - Components::process(): Found all the components!
  WARN root [2019-09-04 20:58:32,042] - TCPPacketReceiver::processPendingEvents(127.0.0.1:56715): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
  INFO root [2019-09-04 20:58:32,044] - Components::removeComponentByChannel: dbmgr : 4000, Normal exit!
